### ISSUE TITLE
Posunutí tlačítka "předchozí číslo" a "následující číslo" výš v pravém panelu

### ISSUE THREAD

**Issue by annie-cz** [ceskaexpedice/kramerius-web-client#763]
Poznámka (a v případě ČDK) zdrojová knihovna čísel periodik může posouvat menších obrazovek tlačítko pro listování na další číslo tak, že je potřeba k němu scrollovat, což není příliš user friendly. Přijde mi, že by nejdůležitejší tlačítka měly být nějak logicky blízko sebe, aby se uživatel nemusel brodit před hromadu přebytečných informací (neříkám, že nejsou důležité, ale ovládací prvky mi přijdou z uživatelského hlediska důležitější). Šlo by to překlikávání se na další a předchozí číslo dát někam výš, třeba hned pod tlačítko "přejít na periodikum"?

Ukázka, ten stejný dokument v ČDK a NK v rámci Digitalniknihovna.cz na poslední verzi klienta. Jsem na 13 palcovém displayi.

https://ceskadigitalniknihovna.cz/uuid/uuid:efd123f0-573a-11e7-a7b7-005056827e51

<img width="1468" alt="image" src="https://github.com/ceskaexpedice/kramerius-web-client/assets/43028501/243b962a-faf9-434b-b6a4-0ddb6acfe2e8">

https://www.digitalniknihovna.cz/nkp/uuid/uuid:efd123f0-573a-11e7-a7b7-005056827e51 

<img width="1470" alt="image" src="https://github.com/ceskaexpedice/kramerius-web-client/assets/43028501/13715abf-4f45-40d3-9fcf-d86619b7fce3">

Okrajově souvisí s #761 , kde se také řeší menší monitory/displaye apod.

**Comment by zabak** [comment]
Souhlasím, navigace v periodiku by přinejmenším měla být nahoře, přinejlepším i přepracovaná, aby byla graficky výraznější.

**Comment by honza-rychtar** [comment]
Byl to jeden z výstupů UX analýzy - přepracování navigace v periodiku
